app/Http/Controllers/PostController.php

```diff
@@ -5,16 +5,16 @@
 use App\Models\Post;
```

Comment:
Fix this file. This shouldn't have any modifications since we're only creating APIs.